builtin/providers/azurerm/resource_arm_local_network_gateway_test.go

```diff
@@ -0,0 +1,102 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/core/http"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMVirtualNetwork_basic(t *testing.T) {
```

Comment:
This needs to be `TestAccAzureRMLocalNetworkGateway_basic` - otherwise there is a duplicate definition preventing build.